bot-components/GitHub_queries.ml

```diff
@@ -682,6 +682,18 @@ let get_base_and_head_checks ~bot_info ~owner ~repo ~pr_number ~base ~head =
                   Error
                     (f "No check suite %d for head commit %s/%s@%s." appId owner
                        repo head )
+              | Some (_ :: _ :: _), _ ->
+                  Error
+                    (f
+                       "Got more than one checkSuite %d for base commit \
+                        %s/%s@%s."
+                       appId owner repo base )
+              | _, Some (_ :: _ :: _) ->
+                  Error
+                    (f
+                       "Got more than one checkSuite %d for base commit \
```

Comment:
Is this meant to say `head` rather than `base`?

Reply:
changed